.gitignore

```diff
@@ -35,6 +35,9 @@ yarn-error.log*
 # vercel
 .vercel
 
+# vercel - force build via yarn by removing package-lock.json
+package-lock.json
```

Comment:
```suggestion
/ath.ooo/package-lock.json
```

Reply:
Does this pertain to just the `ath.ooo` project or all sub-projects in this monorepo now? I see that `ath-og-generator` is built on vercel too :)

Reply:
sorry I just committed it right before I saw your message haha. so the way the monorepos work from vercel is you just tell it to only look at one subfolder and deploy from there (in this case, /ath.ooo/), and then git (and gitignore) are keeping track of the superfolder, so to gitignore something in any of the 3 "subrepos" we'll have to do it like that, where we specify _which_ `package-lock.json` file we're referring to (or whatever file)

Reply:
at least I'm pretty sure that's how it all works, but I could be wrong or missing something. so I assume the gitignore is just so you can use npm if you want and it won't mess up the builds by committing package-locks to the remote repo

Reply:
that's interesting, i'd be down to investigate this further but for now no need to delve in deeply 👍